Assets/Scripts/Augment/AugmentImplementations/RopeBody.cs

```diff
@@ -56,16 +65,25 @@ public override void Start()
     private void PullingWire()
     {
         playerBody.AddForce(-(playerBody.position - rope.CurrentAnchor).normalized * pullForce, ForceMode.Acceleration);
-        if (rope.RopeLength > ropeLength + 4f)
-        {
-            isWired = false;
-            rope.enabled = false;
-            gunController.stats.Ammo = 0;
-            plugAnchor.gameObject.SetActive(false);
-        }
+        if (rope.RopeLength > ropeLength + 4f || (movement.StateIsAir && rope.RopeLength > ropeLength + 1f))
+            RemoveRope();
             
     }
 
+    private void KillRope(HealthController healthController, float damage, DamageInfo info)
```

Comment:
Perhaps let it be an overload of RemoveRope instead, so the naming stays consistent.

Reply:
Fixed 👍